### ISSUE TITLE
the FP of authentic image

### ISSUE THREAD

**Issue by chengxius** [pengzhou1108/RGB-N#19]
hello, peng
      I have some questions while I learn your model.  I tested fifty authentic coco images with  your model , the  confidence of the highest score box of  twenty-four images  is above 0.5, I think the situation as  the FP of authentic image . In our application scenario, we hope  the probability of authentic  image detected as  tampered image can be as low as possible. I have tried many methods to drop the FP, however all failed .I don't know if you have  considered this problem，and how can I handle with the problem?
      Looking forward to your reply ， thank you!

**Comment by Israrulhaq190** [comment]
Hi @chengxius 

1. On how any iterations you have trained the model ? I have also trained the model on 110k iterations.

2. Have you used COCO synthetic dataset for training provided by the author ? 

3. Have you fine tuned on further datasets such as COVER, Columbia, CASIA etc ?

Thanks

**Comment by chengxius** [comment]
> Hi @chengxius
> 
> 1. On how any iterations you have trained the model ? I have also trained the model on 110k iterations.
> 2. Have you used COCO synthetic dataset for training provided by the author ?
> 3. Have you fine tuned on further datasets such as COVER, Columbia, CASIA etc ?
> 
> Thanks

yes, I have tried iterations more than 110k  and kinds of datasets ,but don't work out .

**Comment by Sandy-Zeng** [comment]
Hi @chengxius ,
Have you solved the problem of false positive? Could you please share some solutions? I have also encounter similar problems in my application.

**Comment by Sandy-Zeng** [comment]
Hi @chengxius ,
Have you solved the problem of false positive? Could you please share some solutions? I have also encountered similar problem in my application. 
Thanks!
